galsim/chromatic.py

```diff
@@ -3019,8 +3129,10 @@ def __init__(self, obj, gsparams=None, propagate_gsparams=True):
                     self._obj.deinterpolated, self._gsparams, self._propagate_gsparams)
         else:
             self.deinterpolated = self
-        self.SED = SED(lambda w:math.sqrt(obj.SED(w)), 'nm', '1')
-        self.wave_list = obj.wave_list
+
+    @property
+    def sed(self):
```

Comment:
Similar to above, isn't sqrt_sed = sqrt(original_sed) ?

Reply:
Good catch on these two.  This was a sloppy copy/paste error.  Clearly we're missing a unit test that checks this.